pkg/testfixtures/storage/mysql.go

```diff
@@ -122,7 +122,7 @@ func (m *mySQLTestContainer) RunMySQLTestContainer(t testing.TB) DatastoreTestCo
 	require.NoError(t, err)
 
 	backoffPolicy := backoff.NewExponentialBackOff()
-	backoffPolicy.MaxElapsedTime = 30 * time.Second
+	backoffPolicy.MaxElapsedTime = time.Minute
```

Comment:
This seems to be a source of much flakiness. Increasing this to a minute seems to decrease the flakiness.